client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -71,26 +62,21 @@ const EditCardDetails = React.createClass( {
 	 * @param fields
 	 */
 	mergeCard( card, fields: {} ) {
+		const { name } = card;
+
 		return extend( {}, fields, {
-			name: card.name
+			name
```

Comment:
I don't think using destructuring here brings much value. This method is used to filter card attributes but also to transform card keys so they match what `<CreditCardForm />` expect. Soon we will add the country code as well as the postal code and you won't be able to use destructuring here because their keys will have to be converted from camelcase to snakecase. What do you think?